src/librustc_target/spec/i686_uwp_windows_msvc.rs

```diff
@@ -0,0 +1,24 @@
+use crate::spec::{LinkerFlavor, Target, TargetResult};
+
+pub fn target() -> TargetResult {
+    let mut base = super::windows_uwp_msvc_base::opts();
+    base.cpu = "pentium4".to_string();
+    base.max_atomic_width = Some(64);
+    base.has_elf_tls = true;
+    base.pre_link_args
+        .get_mut(&LinkerFlavor::Msvc).unwrap().push("/LIBPATH:C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\lib\\store".to_string());
```

Comment:
I suppose this is a consequence of not having the proper environment? It's not super aesthetic, but to be fair I have no idea if there's another way of doing so...
Is there a default value otherwise?

Reply:
it doesn't build without this, but if there is a more elegant way to do it, then by all means..
>  Is there a default value otherwise?
Yes, the non-store libs located at `C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\lib`

Reply:
Actually in VS the LIBPATH is already set in the environment: ```LIBPATH=C:\WINDOWS\Microsoft.NET\Framework64\v4.0.30319;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\LIB\amd64;C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\ATLMFC\LIB\amd64;C:\Program Files (x86)\Windows Kits\10\UnionMetadata;C:\Program Files (x86)\Windows Kits\10\References;C:\Program Files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs\14.0\References\CommonConfiguration\neutral;```

So maybe getting the environment variable from Rust would be enough